opentracing-mock/src/test/java/io/opentracing/mock/MockSpanTest.java

```diff
@@ -26,55 +26,59 @@
     @Test
     public void testSetOperationNameAfterFinish() {
         MockTracer tracer = new MockTracer();
-        Span span = tracer.buildSpan("foo").start();
+        Span span = tracer.buildSpan("foo").startManual();
         span.finish();
 
         try {
             span.setOperationName("bar");
             Assert.fail();
         } catch (RuntimeException ex) {
+            // Comment to prevent empty block warning
```

Comment:
Or you can assign to local var and test equals in the next assert statement.

Reply:
Kind of what I meant in [my comment](https://github.com/opentracing/opentracing-java/pull/190#discussion_r141122680)